src/mn_setup.c

```diff
@@ -2452,6 +2454,16 @@ static const char *layout_strings[] = {
     "Flick Stick Southpaw",
 };
 
+static void UpdateRumble(void)
+{
+    I_UpdateRumbleEnabled();
+    I_RumbleMenuFeedback();
+}
+
+static const char *rumble_strings[] = {
```

Comment:
I think I'd like a thermo more than a sequence of percent values.

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1876/commits/7e010e4a6583fb5faa5b04e504a6d41a471c3ba0